operator/pkg/object/objects_test.go

```diff
@@ -390,7 +390,7 @@ spec:
 	for _, tt := range parseYAMLToK8sObjectTests {
 		t.Run(tt.desc, func(t *testing.T) {
 			k8sObj, err := ParseYAMLToK8sObject([]byte(tt.objString))
-			if err != nil {
+			if err == nil {
```

Comment:
I think we can entirely remove this ut since `ParseK8sObjectsFromYAMLManifest` is well tested in differnt places.